planning/behavior_velocity_planner/src/scene_module/intersection/manager.cpp

```diff
@@ -75,6 +75,8 @@ IntersectionModuleManager::IntersectionModuleManager(rclcpp::Node & node)
   ip.intersection_velocity = node.declare_parameter(ns + ".intersection_velocity", 10.0 / 3.6);
   ip.intersection_max_acc = node.declare_parameter(ns + ".intersection_max_accel", 0.5);
   ip.detection_area_margin = node.declare_parameter(ns + ".detection_area_margin", 0.5);
+  ip.detection_area_right_margin = node.declare_parameter(ns + ".detection_area_right_margin", 0.5);
+  ip.detection_area_left_margin = node.declare_parameter(ns + ".detection_area_left_margin", 0.5);
```

Comment:
Could you update yaml file too?

* Note that there are two yaml files, in behavior_velocity_planner and planning_launch.

Reply:
update yaml file https://github.com/autowarefoundation/autoware.universe/pull/1021/commits/35cb232d40f3a96d9d37fc5ada09b21ea72f1cd3